### ISSUE TITLE
Fix /dev/input/event0 getting closed in asteroid-launcher

### ISSUE THREAD

**Issue by atx** [AsteroidOS/meta-smartwatch#52]
/dev/input/event0 occasionally starts feeding asteroid-launcher with EOFs, requiring device restart.

After some debugging, it looks like close() is called on the /dev/input/event0 file descriptor from somewhere completely unrelated (qualcomm driver code most likely). This leads to the file descriptor in the QEvdevTouchScreenHandler class getting replaced with another one, ultimately resulting in getting a bunch of EOFs.

A (very hacky) fix was attempted by firewalling close() using LD_PRELOAD, but as the close()-calling code does not use libc, this does not work. ptrace based firewall would probably work.

Interestingly, restarting asteroid-launcher.service after bootup always results in correctly working touchscreen. Possibly running a dummy Qt application before lipstick could prevent this failure form occuring?

**Comment by FlorentRevest** [comment]
I don't think we have encountered this issue for nearly 10 years so I assume it's probably stale and will close to scrub the backlog
